core/src/main/scala/ai/h2o/sparkling/backend/utils/SharedBackendUtils.scala

```diff
@@ -167,8 +167,6 @@ trait SharedBackendUtils extends Logging with Serializable {
       .add("-flow_dir", conf.flowDir)
       .add("-ice_root", conf.clientIcedDir)
       .add("-port", Some(conf.clientWebPort).filter(_ > 0))
-      .addIf("-network", conf.clientNetworkMask, !RestApiUtils.isRestAPIBased(conf))
```

Comment:
`getH2OWorkerAsClientArgs` is used only in local mode -> no need to set network or ip